app/models/repository/github.rb

```diff
@@ -3,18 +3,29 @@
 require_dependency 'repository/git'
 
 class Repository::Github < ::Repository::Git
+  REPOSITORY_URI_REGEXP = %r{\Ahttps://github\.com/(.+)/(.+)\.git}
+
   has_one :github_credential, foreign_key: :repository_id, dependent: :destroy
   accepts_nested_attributes_for :github_credential
 
   safe_attributes 'access_token', 'webhook_secret'
-  validates_presence_of :url
+  validates :url, presence: true, format: { with: REPOSITORY_URI_REGEXP }
   validates_presence_of :access_token
   validates_presence_of :webhook_secret
 
   delegate :bare_clone, :fetch_remote, :update_remote_url, to: :scm
   after_create :bare_clone
   after_update :update_remote_url
 
+  def self.human_attribute_name(attribute_key_name, *args)
+    if attribute_key_name.to_s == 'url'
```

Comment:
Is it necessary to check `attribute_key_name`?
I think we can delegate everything.
(`superclass.superclass` is a little complicated.)

```ruby
def self.human_attribute_name(attribute_key_name, *args)
  Repository.human_attribute_name(attribute_key_name, *args)
end
```

Reply:
Oh! It's more simple and readable code. Thanks your suggestion.

I pushed https://github.com/agileware-jp/redmine_github/pull/50/commits/062971b833c21bae58385313534b817ff903a237 .

Reply:
I  see. I wrote on https://github.com/agileware-jp/redmine_github/pull/50/commits/f700e4dc5da63afda4356d0ab0e09e5d676540f7 .